src/simulation/simulation_problem_results.jl

```diff
@@ -152,6 +167,10 @@ Set the system in the results instance.
 
 Throws InvalidValue if the system UUID is incorrect.
 """
+function set_system!(results::SimulationProblemResults, system::AbstractString)
```

Comment:
Docstring should say that this is a path to the JSON file.

Reply:
fixed via 21a39e66bd2921ac995a2ac048537fee285b3f41